server/index.ts

```diff
@@ -0,0 +1,19 @@
+import express, { Request, Response, Application } from 'express';
+import bodyParser from 'body-parser';
+require("dotenv").config();
+const app: Application = express();
```

Comment:
TS should infer this type from `express()`, we don't need to declare a type here